scripts/process-data/healthkitv2electrocardiogram.R

```diff
@@ -0,0 +1,216 @@
+# Functions
+
+#' Create i2b2 summaries
+#'
+#' `ecg_stat_summarize()` summarizes the ECG data in a data frame on 
+#' specific time scales (all data and weekly) for the following statistics: 
+#' Number of records.
+#'
+#' @param df A data frame with `participantidentifier`, `concept`, `value`, and 
+#' some combination of `startdate`, `enddate`, `date`, or `datetime` columns.
+#'
+#' @return A data frame that differs in size from `df`. The original `value` 
+#' column is dropped and replaced by a new `value` column containing the values 
+#' of each summary statistic computed for each group of data (group by 
+#' `participantidentifier` and `concept` for all data, and 
+#' `participantidentifier`, `concept`, `year`, `week` for weekly summaries).
+ecg_stat_summarize <- function(df) {
```

Comment:
Is this identical to the Fitbit version? If so, it would lend strength to my previous suggestion to make this a generalized function.

Reply:
Same reply as above